__tests__/actions/form.js

```diff
@@ -20,18 +31,15 @@ describe('actions > api > form', () => {
           autoPlan: true,
           debouncePlanTimeMs: 500
         },
-        currentQuery: {
-          from: { lat: 12, lon: 34 },
-          to: { lat: 34, lon: 12 }
-        },
+        currentQuery: queries[0],
         searches: []
       }
     }
 
-    const formChangedAction = formChanged()
-
     const mockDispatch = jest.fn()
     for (var i = 0; i < 10; i++) {
+      // Alternate back and forth between two queries on each iteration.
+      const formChangedAction = formChanged(queries[i % 2], queries[(i + 1) % 2])
```

Comment:
This test can definitely use a refactor. It seems like no debouncing is happening anymore due to what the snapshot looks like. It may make sense to comment this out again and reassess later.

Reply:
Hmm. Ok -- you think we should rename the file again or just comment out the code?

Reply:
Let's comment it all out for now. I initially put this test case in here to verify the functionality when developing the CFAZ react native app because it always auto-plans trips. But that was a while ago and things have drifted in the codebase, so I'm fine with removing the test case.